server/plugin/command_test.go

```diff
@@ -71,24 +71,28 @@ func TestPluginExecuteCommand(t *testing.T) {
 				Name:        "setting-anonymous",
 				Type:        "bool",
 				Placeholder: "Don't show who voted for what when the poll ends",
+				Default:     "false",
```

Comment:
Would it be possible for you to add a few tests for cases where some default settings is enabled?